objects.txt

```diff
@@ -1,2202 +0,0 @@
-aardvark
-aardwolf
-accelerator
-accordion
-account
-acoustic
```

Comment:
oh, hold on ...  looks like objects.txt may have accidentally gotten deleted?

Reply:
there's (still) words/objects.txt. this file appears to have been an accidental dupe.

Reply:
ah I see, you're right.  Cool.